conda_forge_tick/git_utils.py

```diff
@@ -739,7 +1018,13 @@ def get_remote_url(
 
         return super().get_remote_url(upstream_owner, repo_name, connection_mode)
 
-    @lock_git_operation()
+    def push_to_repository(
+        self, owner: str, repo_name: str, git_dir: Path, branch: str
+    ):
+        logger.debug(
+            f"Dry Run: Pushing changes from {git_dir} to {owner}/{repo_name} on branch {branch}."
```

Comment:
see below, should we add this into a `debug_output` method?

Reply:
done